assembl/indexing/utils.py

```diff
@@ -134,6 +134,10 @@ def get_data(content):
         if not data['idea_id']:
             return None, None
 
+        # If the post is a fiction for Bright Mirror, don't index it.
+        if Idea.get(data['idea_id'][0]).message_view_override == 'brightMirror':
```

Comment:
```from assembl.models.timeline import Phases
if Idea.get(data['idea_id'][0]).message_view_override == Phases.brightMirror.value:
...
```

Reply:
I didn't read this one first, no this is wrong, you shouldn't use the Phases enum for message_view_override, even if the string is the same here. Create a new enum for message_view_override in this case.